### DISCUSSION TITLE
Sound bug.

### DISCUSSION THREAD

**Discussion by youhim** [jpochyla/psst#470] [General]
When you’re working on a computer, the application’s sound bubbles, stutters, disappears. With any interaction with the interface. It’s impossible to listen.

**Comment by Insprill** [comment]
This is a known issue in one of our dependencies: https://github.com/jpochyla/psst/issues/235. The fix is to assign psst high/highest priority in Task Manager or use a tool to do this automatically.

**Reply by youhim** [reply]
Since 2021, there's still no way to fix this? The bug is critical.

**Reply by jacksongoode** [reply]
Is this still an issue with you on the latest version?

**Reply by youhim** [reply]
> Is this still an issue with you on the latest version?

Unfortunately, yes.

**Reply by jacksongoode** [reply]
Are you on Windows? Some info about your platform would be good. I can't see this issue on Linux or macOS.

**Reply by youhim** [reply]
> Are you on Windows? Some info about your platform would be good. I can't see this issue on Linux or macOS.

Windows 10x64 Enterprise 22H2
Intel(R) Core(TM) i3-2100 CPU @ 3.10GHz   3.10 GHz
16,0 Gb

**Comment by SuisChan** [comment]
What audio backend are you using? If cpal, then try cubeb, the difference may be noticeable
